spec/latest/core/core.jsonld

```diff
@@ -505,6 +507,22 @@
       "domain": "hydra:IriTemplateMapping",
       "range": "xsd:string",
       "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:skip",
+      "@type": "rdf:Property",
+      "label": "skip",
+      "comment": "Instructs to skip N elements of the set.",
+      "range": "xsd:integer",

```

Comment:
What's the domain of those properties?

Reply:
shall we add "domain": "hydra:IriTemplateMapping"?

Reply:
No domain is specified, enabling any kind of resource to have that property. It's purpose it to have a property that could be used indeed in IriTemplateMapping's _property_ relation. Domain of `hydra:IriTemplateMapping` would be incorrect here.